spacepy/toolbox/__init__.py

```diff
@@ -1124,17 +1134,26 @@ def get_url(url, outfile=None, reporthook=None, cached=False,
         An established http connection (HTTPS is also okay) to use with
         ``keepalive``. If not provided, will attempt to make a connection.
 
+    Raises
+    ------
+    ValueError
+        If conn is provided and keepalive is not True
+    RuntimeError
+        HTTP status >= 400 returned
+    RuntimeError
+        If cached is True and an outfile is not set.
```

Comment:
> 
>     1. The cached `True` without `outfile` should probably be a `ValueError`...yes that's how I wrote it but documenting it like this, it seems clearly wrong.

Should I update that here? Can without issue. Likely justifies a change entry in the docs. 

> 
>     2. The numpydoc guide isn't very clear on this, but I think it should probably be one entry per exception type, and will probably take some playing to get the formatting right. So (assuming 1) "If conn is provided and keepalive is False, or cached is True and no outfile provided." Might also be nice to have them alphabetical by exception type.

I feel like the guide examples double up on exception type and in the rendering I feel like it makes the most sense. Otherwise the doc has to say something like "bla bla or bla bla" fine for short messages but tough for long ones. 

> 
>     3. numpydoc guide says "This section should be used judiciously, i.e., only for errors that are non-obvious or have a large chance of getting raised." I think it should be obvious that putting in invalid combinations of arguments will result in errors (the parameters section should probably make invalid combinations more apparent, though.) The HTTP error is _maybe_ non-obvious, but it's pretty clear what happened if it is raised. Do we want to have a house style to always have this? At a certain point it degenerates into the Java "throws" declarations...

Fair enough. Drop this one as the message is clear?
```
    RuntimeError
        HTTP status >= 400 returned
```

Reply:
Updated, have a look after the push. I think well. Dropped the Raises in favor of notes in Parameters and did update to `ValueError`